stories/index.stories.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
```

Comment:
Can it be tested right now? Or do we need an additional html file to test it?

Reply:
Was going to ask the same.

Reply:
It can be tested right now see my instructions in another comment